Canvas.sln.DotSettings

```diff
@@ -0,0 +1,7 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
> That's what I mean. I would add rules to tell resharper that the current style is the correct one

Sorry but I don't meant that we should add to the repo that file `Canvas.sln.DotSettings`.

We don't want to have 3rd party tooling-specific config settings on the repo. 

Can you please undo that?